Framework/PythonInterface/plugins/algorithms/IndirectTransmission.py

```diff
@@ -173,7 +173,7 @@ def _get_efixed(self, workspace):
         except ValueError:
             # If that fails then get it by taking from group of all detectors
             wsHandle = mtd[workspace]
-            spectra_list = list(range(0, wsHandle.getNumberHistograms()))
+            spectra_list = list(range(1, wsHandle.getNumberHistograms() + 1))
```

Comment:
In this case yes, I was going on the comment that it takes a group of *all* detectors whereas previously it only did `1...n-1` detectors. For that reason I changed it to end use all the spectra which may change behaviour slightly.